src/modules/cmdpal/ext/Microsoft.CmdPal.Ext.WindowsSettings/Pages/FallbackWindowsSettingsItem.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.CmdPal.Ext.WindowsSettings.Classes;
+using Microsoft.CmdPal.Ext.WindowsSettings.Commands;
+using Microsoft.CmdPal.Ext.WindowsSettings.Helpers;
+using Microsoft.CmdPal.Ext.WindowsSettings.Properties;
+using Microsoft.CommandPalette.Extensions.Toolkit;
+
+namespace Microsoft.CmdPal.Ext.WindowsSettings.Pages;
+
+internal sealed partial class FallbackWindowsSettingsItem : FallbackCommandItem
+{
+    private readonly Classes.WindowsSettings _windowsSettings;
+
+    private readonly string _title = Resources.settings_fallback_title;
+    private readonly string _subtitle = Resources.settings_fallback_subtitle;
+
+    public FallbackWindowsSettingsItem(Classes.WindowsSettings windowsSettings)
+        : base(new NoOpCommand(), Resources.settings_title)
+    {
+        Icon = IconHelpers.FromRelativePath("Assets\\WindowsSettings.svg");
```

Comment:
nit: i've found the best practice is to have a single `static IconInfo` instance for icons from paths. See things like `ext\Microsoft.CmdPal.Ext.Shell\Icons.cs`. That way we don't keep allocating more objects for the same icon